critiquebrainz/frontend/external/musicbrainz_db/tests/release_group_test.py

```diff
@@ -54,3 +55,29 @@ def test_fetch_release_groups(self):
         self.assertEqual(len(release_groups), 2)
         self.assertEqual(release_groups['7c1014eb-454c-3867-8854-3c95d265f8de']['title'], 'Numb/Encore')
         self.assertEqual(release_groups['8ef859e3-feb2-4dd1-93da-22b91280d768']['title'], 'Collision Course')
+
+
+    def test_fetch_browse_release_groups(self):
+        self.browse_release_groups_query = self.mock_db.query.return_value.options.return_value.join.return_value.\
+            join.return_value.join.return_value.join.return_value.filter.return_value.filter.return_value
```

Comment:
This is one case where it's worth splitting an infinitely deep attribute selection into multiple variables and/or function calls. I have no idea what's going on here.